.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Bug Report
+about: Report problems and bugs in Sputnik
+title: ''
+labels: 'bug'
+assignees: ''
+
+---
+
+**Describe the issue**
+A clear and concise description of what the issue is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Environment**
+
+- OS: [e.g. OSX 10.13.4, Windows 10]
```

Comment:
Maybe removing Windows would make sense because I think Sputnik is actually mac-only (correct me if I'm wrong 🙂)

Reply:
It would appear so for now! I've revised that, and added a spot for which kind of cpu architecture, silicon/x86.